src/flux/task.coffee

```diff
@@ -117,6 +117,15 @@ TaskConfig =
       allSteps = getSteps(@_steps[taskId])
       steps = getCompleteSteps(allSteps)
 
+    # return a feedback message with varying amounts of praise for the number of questions correct.
+    # Used by practice and readings.  The number of questions correct is not shown for homeworks
+    getFeedbackMessage: (taskId) ->
+      incorrect = _.where(this._steps[taskId], is_correct: false).length
+      switch incorrect
+        when 0 then "Great job, all questions answered correctly!"
+        when 1 then "Good job, only 1 question incorrect!"
+        else "#{incorrect} questions were incorrect, try harder next time"
```

Comment:
No, almost certainly not - that's really just a placeholder until Fred gets proper verbiage.